app/Policies/EpistolaPolicy.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Policies;
+
+use App\Models\Role;
+use App\Models\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class EpistolaPolicy
+{
+    use HandlesAuthorization;
+
+    public function create(User $user)
+    {
+        return $user->hasRoleBase(Role::COLLEGIST);
+        //return $user->hasRoleWithObjectNames(Role::STUDENT_COUNCIL, ['communication-leader', 'communication-member']);
```

Comment:
Probably we want to use the commented out policy?

Reply:
I'm not sure. The comm. leader will go through it anyway before sending, and I'd like that if we could get the collegists fill this with content.
But I should add a delete function.